### PR TITLE
A better way to deploy, IMO. Including documentation.

### PR DESCRIPTION
**Pull request by JaapMoolenaar**
I changed the method of deploying, mainly because I wanted to deploy more easily (without having to find and type/copy the code every time). The upload code can be set in the .deploykey file.

I also needed a way to ignore certain files, because coffeescripts get autocompiled, leaving invalid .js files in the same folder. The files can be ignored using the .deployignore file.

I also removed the powershell script, windows users should just use git for windows, which works perfectly fine.

To accompany the changes, i rewrote the readme file.

### CODE REVIEW

**Comment by EmielM** [comment]
Thanks for your work!

Can the windows Git console execute the deploy file correctly? We've noticed a lot of aspiring developers still use Windows.

The possibility of putting your key in a separate deploykey file seems reasonable, we'll be sure to add this.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,34 @@
...
+
+Paste your upload code in the `.deploykey` file.
+
+In case you don't want to upload certain files, edit the `.deployignore` and add each file on a new line (wildcards are supported using an asterisk *, eg. *.psd). This file already holds the _.deploykey_ file, so that your upload code won't be sent to the plugin servers.
+
+All that's left now is running `./deploy`
+
+The manifest file
+-----------------
+The manifest file holds configuration for you app. You can set things like `name`, `description` and `icon` here (a comprehensive list of all available icons will be made available shortly)
```
Comment (lesteenman) [inline README.md:34]: Yea, that comprehensive list would surely come in useful ;)

**Comment by JaapMoolenaar** [comment]
Hi @EmielM,

I'm one of those (aspiring) developers ;-) mainly because I develop a lot using sql server for persistence, it just so happens that support for (connecting with) sql server on unix based systems is (almost) non existent.

The script runs nicely, however, I made an adjustment to the cUrl line. I don't use a temporary file (in memory) but an actual file (on disk), like the wget method (and DRYd it). This because windows tends to not handle it correctly, and errors out. The above method work every time.

@dolor, I snuck that right in there, didn't I? ;-) _I would actually also very much like (more) documentation on the db/obs modules_

**Comment by JaapMoolenaar** [comment]
I added a link to a simple zip.exe for windows users.
In case they don't have it (yet).

**Comment by EmielM** [comment]
Merged in history branch (and updated the master branch), thanks! I've reworded README and am a bit hesitant linking to an externally hosted zip.exe for now ;). Perhaps we'll add this later.

Could you verify it's still working?

**Comment by JaapMoolenaar** [comment]
Yes it seems to be working, I do however get a 504 time out instead of the `Installed 1 client script(s), 0 client resource(s) and 1 server script(s)`, even though the files seems to have uploaded correctly (on the second deploy it seems, weird, I'll setup a VM and run it from there to see if a fresh windows install does work...).

That is, using a fresh clone from the example repo and a new group app. When I however use the script on the project I'm working on, everything seems to work fine.

**Comment by EmielM** [comment]
The 504 errors are now resolved.

**Pull request closed without merging**